docs/ref/models/database-functions.txt

```diff
@@ -87,6 +87,23 @@ Usage examples::
     >>> now = timezone.now()
     >>> Coalesce('updated', Cast(now, DateTimeField()))
 
+``Collate``
+-----------
+
+.. class:: Collate(expression, collation)
+
+Takes an expression and a collation name to query against.
+For example, to filter case-insensitively in SQLLite:
+
+    >>> Employee.objects.filter(first_name=Collate(Value('john'), 'nocase'))
+    <QuerySet [<Employee: John>, <Employee: john>]>
+
+It can also be used when ordering, for example with PostgreSQL:
+
+    >>> Employee.objects.order_by(Collate(F('first_name'), 'et-x-icu'))
```

Comment:
Not sure which one we want to document but since string literals tentatively resolve to field names you can drop the `F` wrapping

```suggestion
    >>> Employee.objects.order_by(Collate('first_name', 'et-x-icu'))
```